README.md

```diff
@@ -116,6 +116,29 @@ the matching jobs will not influence the resulting outcome of the
 whole matrix and this action will "see" that as a success (provided
 that the jobs that are not allowed to fail succeed, of course).
 
+```yaml
+# some fields were removed from example to simplify it
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+    matrix:
+       os:
+       - ubuntu-latest
+       - macos-11
+    steps:
+    - run: echo foo
+      continue-on-error: ${{ matrix.os == 'macos-11' }}
```

Comment:
Was it intentional that it's applied on the step level? I feel like it's clearer when this setting is used on the job level, above steps. This is what I was aiming for, at least, in that paragraph above.